tests/modules/word_test.cpp

```diff
@@ -276,3 +277,333 @@ TEST_F(WordTest, returnsFirstElementWhenNoLengthMatch)
 
     ASSERT_TRUE(result == "three");
 }
+class WordTestLocale : public TestWithParam<Locale>
+{
```

Comment:
Please use parametrized tests with TEST_P and use locale param within test like in book module tests